build.cfc

```diff
@@ -181,19 +183,20 @@ component accessors="true" {
                         tokens,
                         cfformat.mergedSettings(reference[setting].example.settings)
                     );
-                    output.append(
-                        formatted
-                            .replace(chr(13), '', 'all')
-                            .reReplace('//\s?', '// #setting#: #isNull(v) ? 'null' : serializeJSON(v)#')
-                            .trim()
-                    );
+                    output[ v ] = formatted
+                        .replace(chr(13), '', 'all')
+                        .reReplace('//\s?', '// #setting#: #isNull(v) ? 'null' : serializeJSON(v)#')
+                        .trim();
                 }
 
-                examples[setting] = output.toList(lf & lf);
+                examples[setting] = output;
             }
         }
 
-        JSONService.writeJSONFile(dir & 'data/examples.json', examples);
```

Comment:
@elpete What was the reason for this change? I am pretty sure the JSONService pretty prints by itself?

Reply:
The JSONService automatically sorts keys.  If you do not pass in a sorting strategy it uses a CommandBox setting.  This is what was causing the keys to not be written in order.  I copied in how it is done in the JSONService to work around this.

Reply:
Ah, thanks! I had forgotten that it won't sort keys if they aren't sorted on disk, but it will if the file doesn't exist at all (by default).